common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RetryingBlockFetcher.java

```diff
@@ -215,8 +227,11 @@ public void onBlockFetchFailure(String blockId, Throwable exception) {
           if (shouldRetry(exception)) {
             initiateRetry();
           } else {
-            logger.error(String.format("Failed to fetch block %s, and will not retry (%s retries)",
-              blockId, retryCount), exception);
+            if (errorHandler.shouldLogError(exception)) {
+              logger.error(
+                  String.format("Failed to fetch block %s, and will not retry (%s retries)",
```

Comment:
Nit: "Failed to fetch block" => "Failed to move block" or "Failed to fetch/upload block

Reply:
And in line 176 the "Retrying fetch" should be fixed, too.

Reply:
Would it be OK to make these changes in the follow up PR?
The RetryingBlockFetcher is filled with many references to "fetch".
I think it would be better to consolidate all these changes in the follow up PR, which focuses just on this name change.

Reply:
Yes, a follow up PR is totally fine.